scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.js

```diff
@@ -46,13 +46,21 @@ export function ContentProfileSchemaEditor(gettext, content) {
             scope.loading = true;
             scope.model.schema = angular.extend({}, content.contentProfileSchema);
             scope.model.editor = angular.extend({}, content.contentProfileEditor);
+            scope.schemaKeys = null;
 
             content.getTypeMetadata(scope.model._id).then((typeMetadata) => {
                 scope.model.schema = angular.extend({}, typeMetadata.schema);
                 scope.model.editor = angular.extend({}, typeMetadata.editor);
                 scope.loading = false;
+                getSchemaKeys();
             });
 
+            const getSchemaKeys = () => {
```

Comment:
Can you please add some documentation? It's hard to say what this function is doing...